README.md

```diff
@@ -108,6 +108,14 @@ Tangle-accelerator supports several different build time options.
 * Docker images
 * MQTT connectivity
 * External database
+* Debug Mode
```

Comment:
Can you mention how would tangle-accelerator react in `debug mode` and `profiling mode`?